frontend/test/metabase/scenarios/visualizations/bar_chart.cy.spec.js

```diff
@@ -103,4 +103,63 @@ describe("scenarios > visualizations > bar chart", () => {
       cy.get(".value-labels").should("contain", "19").and("contain", "20.0M");
     });
   });
+
+  describe("with x-axis series", () => {
+    beforeEach(() => {
+      visitQuestionAdhoc({
+        display: "bar",
+        dataset_query: {
+          type: "query",
+          query: {
+            "source-table": ORDERS_ID,
+            aggregation: [["count"]],
+            breakout: [
+              ["field", PEOPLE.SOURCE, { "source-field": ORDERS.USER_ID }],
+              [
+                "field",
+                PRODUCTS.CATEGORY,
+                { "source-field": ORDERS.PRODUCT_ID },
+              ],
+            ],
+          },
+          database: SAMPLE_DB_ID,
+        },
+      });
+
+      cy.findByText("Settings").click();
+      sidebar().findByText("Data").click();
+    });
+
+    it("should allow you to change the order of the columns", () => {
+      cy.findAllByTestId(/draggable-item/)
+        .eq(0)
+        .trigger("mousedown", 0, 0, { force: true })
```

Comment:
Looks similar to https://github.com/metabase/metabase/pull/25461/files#diff-60057b0c5b15d72015a5f04da49edd11e3c2ec9f41374f80b4c69fb39f0e8964R71
Maybe it worth creating a shared helper for that